packages/calcite-components/src/components/accordion/readme.md

```diff
@@ -29,4 +29,6 @@ For comprehensive guidance on using and implementing `calcite-accordion`, refer
 
 ---
 
+<!-- TODO: How do I change these auto generated stencil links? -->
```

Comment:
These files are no longer generated so you can delete the links for now. We will need to remove the component readmes eventually since they will become outdated